src/pages/recovery/recovery.styles.js

```diff
@@ -2,50 +2,49 @@ import { makeStyles } from '@material-ui/core/styles';
 
 export const useStyles = makeStyles((theme) => ({
   emailInput: {
+    '& h4': {
+      fontSize: '34px',
+      lineHeight: '46px'
+    },
+    '& label': {
+      fontWeight: 400,
+      fontSize: '16px',
+      transform: 'translate(14px, 12px) scale(1)'
+    },
+    '& div > input': {
+      padding: '11px 14px !important',
```

Comment:
remove important, use inputProps for textField component or check child classes for input

Reply:
Removed important.